### PR TITLE
CLI

### PR DESCRIPTION
**Pull request by jeroenrinzema**
This PR introduces a standard CLI which could be used in most situations. The CLI currently consists only out of the run command but could be extended in the future.

### CODE REVIEW

**Comment by velp** [comment]
I know that we are not using such structure in our private repos but will be better to use in Open Source project :)

File: cmd/maestro/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+### Using a Package Manager (Preferred)
+
+> 🚧 Currently it is only possible to download + build the CLI from source
+
+### Downloading a Release from GitHub
+
+> TLDR; one-liner where `<version>` is the full semantic version, e.g., `1.17.0`.
+
+```
+VERSION=0.0.0; curl -sL "https://github.com/jexia/maestro/releases/download/v$VERSION/maestro-$VERSION-linux-amd64.tar.gz" | tar -xzv; chmod +x ./maestro
```
Comment (velp) [inline cmd/maestro/README.md]: In the future will be better to use https://github.com/goreleaser/goreleaser with https://github.com/goreleaser/godownloader to create cross-platform releases (especially for the client) and to create shell script as golangci-lint has (https://github.com/golangci/golangci-lint/blob/master/install.sh).

After that, we can provide something like this `curl -sfL https://raw.githubusercontent.com/jexia/maestro/master/install.sh | sh -s -- -b ${GOPATH}/bin <VERSION>`

Comment (jeroenrinzema) [reply cmd/maestro/README.md]: Agreed, this is on the road map to be introduced once `v0.1.0` is released. Ideally are the binaries build and included when a release is created.

Comment (velp) [reply cmd/maestro/README.md]: goreleaser creates release automatically, so you can just add it as an additional step to CI pipeline and when you will create a tag goreleaser will create all of binaries and release

**Pull request merged**